tests/test_tlsfuzzer_helpers.py

```diff
@@ -13,7 +13,8 @@
     from unittest.mock import call
 
 
```

Comment:
The commit text is quite silent on why is that needed. The next commit clarifies it though.